contracts/basset_vault/src/tests/mod.rs

```diff
@@ -213,7 +213,13 @@ impl WasmMockQuerier {
             }
             QueryRequest::Wasm(WasmQuery::Smart { contract_addr, msg }) => {
                 let response = match self.wasm_query_smart_responses.get(contract_addr) {
-                    Some(response) => response,
+                    Some(responses_map) => {
+                        if responses_map.len() == 1 {
+                            responses_map.values().next().unwrap()
+                        } else {
+                            responses_map.get(msg).unwrap()
+                        }
```

Comment:
This looks unclear to me. Why don't you use `get` if there is only one element in the map? Maybe leave comment for this case

Reply:
Cause `msg` in that case is not the same is what in `key` in that map.

Reply:
look at this line: https://github.com/Nexus-Protocol/basset-vault-contracts/pull/41/files#diff-b18523bc232b3ef855710e8279b40a8961ed3c1e778a3dd20778112a0c75a596R491

Reply:
Maybe add comment to both places. Btw it isn't that important, it was just unclear for me, so I considered that it would be easier to read the code if there was a comment

Reply:
Comment added